src/shared_modules/content_manager/tests/unit/cleanUpContent_test.cpp

```diff
@@ -40,5 +40,5 @@ TEST_F(CleanUpContentTest, OutputPathSet)
 
     // Check the folder content
     EXPECT_TRUE(std::filesystem::is_empty(DOWNLOAD_DIR));
-    EXPECT_FALSE(std::filesystem::is_empty(CONTENTS_DIR));
+    EXPECT_TRUE(std::filesystem::is_empty(CONTENTS_DIR));
```

Comment:
Comment: the module didn't delete the folder before because the router published the files. Now, the callback reads the data and after, it's safe to remove them.